src/widgets/list/lv_list.h

```diff
@@ -76,6 +76,35 @@ const char * lv_list_get_button_text(lv_obj_t * list, lv_obj_t * btn);
  */
 void lv_list_set_button_text(lv_obj_t * list, lv_obj_t * btn, const char * txt);
 
+#if LV_USE_TRANSLATION
+
+/**
+ * Add translation tag text to a list
+ * @param list      pointer to a list, it will be the parent of the new label
+ * @param tag       translation tag of the new label
+ * @return          pointer to the created label
+ */
+lv_obj_t * lv_list_add_text_translation_tag(lv_obj_t * list, const char * tag);
+
+/**
+ * Add translation tag button to a list
+ * @param list      pointer to a list, it will be the parent of the new button
+ * @param icon      icon for the button, when NULL it will have no icon
+ * @param tag       translation tag of the new button, when NULL no translation tag will be added
+ * @return          pointer to the created button
+ */
+lv_obj_t * lv_list_add_button_translation_tag(lv_obj_t * list, const void * icon, const char * tag);
+
+/**
+ * Set translation tag text of a given list button
+ * @param list      pointer to a list
+ * @param btn       pointer to the button
+ * @param tag       pointer to the translation tag
+ */
+void lv_list_set_button_text_translation_tag(lv_obj_t * list, lv_obj_t * btn, const char * tag);
```

Comment:
```suggestion
void lv_list_set_button_translation_tag(lv_obj_t * list, lv_obj_t * btn, const char * tag);
```

On other places we use either "text" or "translation_tag" as well.

Reply:
done!